pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/LetterTemplate.kt

```diff
@@ -88,30 +102,42 @@ sealed class Element<out Lang : LanguageSupport> {
     class NewLine<out Lang : LanguageSupport> : Element<Lang>()
 
     sealed class Text<out Lang : LanguageSupport> : Element<Lang>() {
-        data class Literal<out Lang : LanguageSupport> private constructor(private val text: Map<Language, String>) :
-            Text<Lang>() {
+        data class Literal<out Lang : LanguageSupport> private constructor(
+            private val text: Map<Language, String>,
+            val fontType: FontType
+        ) : Text<Lang>() {
 
             fun text(language: Language): String =
                 text[language]
                     ?: throw IllegalArgumentException("Text.Literal doesn't contain language: ${language::class.qualifiedName}")
 
             companion object {
-                fun <Lang1 : Language> create(lang1: Pair<Lang1, String>) =
-                    Literal<LanguageSupport.Single<Lang1>>(mapOf(lang1))
+                fun <Lang1 : Language> create(
+                    lang1: Pair<Lang1, String>,
+                    fontType: FontType = FontType.PLAIN
+                ) = Literal<LanguageSupport.Single<Lang1>>(mapOf(lang1), fontType)
 
                 fun <Lang1 : Language, Lang2 : Language> create(
                     lang1: Pair<Lang1, String>,
                     lang2: Pair<Lang2, String>,
-                ) = Literal<LanguageSupport.Double<Lang1, Lang2>>(mapOf(lang1, lang2))
+                    fontType: FontType = FontType.PLAIN,
+                ) = Literal<LanguageSupport.Double<Lang1, Lang2>>(mapOf(lang1, lang2), fontType)
 
                 fun <Lang1 : Language, Lang2 : Language, Lang3 : Language> create(
                     lang1: Pair<Lang1, String>,
                     lang2: Pair<Lang2, String>,
                     lang3: Pair<Lang3, String>,
-                ) = Literal<LanguageSupport.Triple<Lang1, Lang2, Lang3>>(mapOf(lang1, lang2, lang3))
+                    fontType: FontType = FontType.PLAIN,
+                ) = Literal<LanguageSupport.Triple<Lang1, Lang2, Lang3>>(mapOf(lang1, lang2, lang3), fontType)
             }
         }
 
+        enum class FontType {
```

Comment:
Vil vi noen gang ha behov for bold og italic? Vet at CSS har font-weight for tykkelse og font-style for kursiv.

Reply:
Vi har allerede behov i vedlegg (rettigheter og plikter) for begge deler.

Reply:
Jeg mente **_samtidig_**